src/infrastructure/sanity/clients/ContentClient.ts

```
@@ -0,0 +1,164 @@
+import { FetchByIdOptions } from '../interfaces/FetchByIdOptions'
+import { ConfigProvider } from '../interfaces/ConfigProvider'
+import { QueryExecutor } from '../interfaces/QueryExecutor'
+
+/**
+ * ContentClient extends SanityClient with content-specific methods
+ * for easier content fetching and management
+ */
+export class ContentClient extends SanityClient {
+  constructor(configProvider?: ConfigProvider, queryExecutor?: QueryExecutor) {
+    super(configProvider, queryExecutor)
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)